ingest/Snakefile

```diff
@@ -57,12 +54,13 @@ rule all:
         _get_all_targets,
```

Comment:
Ditto missed this when I was reviewing https://github.com/nextstrain/zika/commit/cee62cf88bacd0c92ccaa0f5b231b1268a413514, but I think the `all` rule can be simplified to the default targets and the entire `_get_all_targets` function can be removed. 

```
rule all:
    input:
        expand(["results/sequences_{serotype}.fasta", "results/metadata_{serotype}.tsv"], serotype=serotypes)
```